src/display/display_utils.js

```diff
@@ -562,6 +562,8 @@ function getRGB(color) {
 function getColorValues(colors) {
   const span = document.createElement("span");
   span.style.visibility = "hidden";
+  // NOTE: The following does *not* affect `forced-colors: active` mode.
+  span.style.colorScheme = "light";
```

Comment:
Maybe we should use `only light` here:
https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme#only

Reply:
Sure, fixed now.